lib/fizzy/parser.hpp

```diff
@@ -64,16 +70,16 @@ inline parser_result<Limits> parse_limits(const uint8_t* pos)
 }
 
 template <typename T>
-parser_result<std::vector<T>> parse_vec(const uint8_t* pos)
+parser_result<std::vector<T>> parse_vec(const uint8_t* pos, const uint8_t* end)
```

Comment:
Other functions are `inline`

Reply:
Mostly `inline` is required for functions and full template specializations if they are in headers. I was also using `inline` for full template specializations in `.cpp` file to check if used anywhere else. I will make this `inline` as well.